ci/prebuild/prebuild.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+if [[ "$TRAVIS_OS_NAME" == "linux" ]];
+then
+    unset OPENCV4NODEJS_DISABLE_AUTOBUILD
```

Comment:
Have you actually tried this out? I can imagine, building opencv from scratch for every single build task in the matrix could result in very long build times.

Reply:
In my fork I'm only running prebuild tasks and yes, the job takes quite some time.
Funny enough, building OpenCV from scratch for each Linux job is (on average) actually faster than installing OpenCV@3 via Homebrew for macOS builds.
![Screenshot 2019-03-13 at 09 31 56](https://user-images.githubusercontent.com/7813765/54264776-1d344b00-4574-11e9-9c33-e28cfb6160e1.png)

Reply:
Oh okay, ~20min is actually quite good. I think that should be fine.